RecoTracker/LST/BuildFile.xml

```diff
@@ -0,0 +1,10 @@
+<use name="DataFormats/Common"/>
+<use name="alpaka"/>
+<use name="DataFormats/TrackerRecHit2D"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<use name="RecoTracker/LSTCore"/>
+<flags CXXFLAGS="-DLST_IS_CMSSW_PACKAGE"/>
```

Comment:
How is this flag used ?
I don't seem to find it anywhere in the code.

Reply:
This was an old flag that we were using for the standalone and somehow we forgot to remove it from `BuildFile.xml`